### PR TITLE
loggedIn instead of loadOnStart

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

**Comment by zapu** [comment]
looks like this RPC is being called once when starting the app while already being logged in (as expected); but twice when logging in after app has started in unlogged state or user has logged out.

**Comment by zapu** [comment]
also definitely just ignore any errors coming from that RPC - no reason to see black bars because you happened to be offline

**Comment by songgao** [comment]
@chrisnojima ping 🏓

**Comment by songgao** [comment]
> Can't we keep the action but change the phase. We don't need this on start

@chrisnojima  There's `reloggedIn` but it looks like it's not intended to trigger when app starts with user already logged in. I guess we could add a phase for that. Could you elaborate a bit how and why the extra action dispatch (`loadOnStart`) helps?

**Comment by chrisnojima** [comment]
i think the original action should stay and original phase, just ignore if not logged in

File: shared/actions/config/index.tsx [hunk 1]
```diff
@@ -562,7 +562,7 @@ const updateServerConfig = async (_: Container.TypedState, action: ConfigGen.Loa
   action.payload.phase === 'startupOrReloginButNotInARush' &&
```
Comment (chrisnojima) [inline shared/actions/config/index.tsx]: no i mean check state.config.loggedIn here

**Pull request merged**